client/components/inline-support-link/context-links.js

```diff
@@ -400,6 +400,54 @@ const contextLinks = {
 		link: 'https://wordpress.com/support/check-your-sites-performance/',
 		post_id: 386816,
 	},
+	'manage-your-dns-records': {
+		link: 'https://wordpress.com/support/domains/custom-dns/',
+		post_id: 6595,
+	},
+	'dns-default-mx-records': {
+		link: 'https://wordpress.com/support/domains/custom-dns/view-or-restore-default-dns-records/#default-mx-records',
+		post_id: 386585,
+	},
+	'dns-default-a-records': {
+		link: 'https://wordpress.com/support/domains/custom-dns/view-or-restore-default-dns-records/#default-a-records',
+		post_id: 386585,
+	},
+	'dns-default-cname-records': {
+		link: 'https://wordpress.com/support/domains/custom-dns/view-or-restore-default-dns-records/#default-cname-record',
+		post_id: 386585,
+	},
+	'edit-or-delete-dns-records': {
+		link: 'https://wordpress.com/support/domains/custom-dns/edit-or-delete-dns-records/',
+		post_id: 386584,
+	},
+	'add-a-new-dns-record': {
+		link: 'https://wordpress.com/support/domains/custom-dns/add-a-new-dns-record/',
+		post_id: 386583,
+	},
+	'edit-or-delete-dns-record': {
+		link: 'https://wordpress.com/support/domains/custom-dns/edit-or-delete-dns-records/',
+		post_id: 386584,
+	},
```

Comment:
I noticed the links `edit-or-delete-dns-records` and `edit-or-delete-dns-record` are the same link.
Should we just use one?

Reply:
Thanks for catching this. I removed the obsolete entry.